.docker/Dockerfile.ckan

```diff
@@ -15,13 +16,14 @@ RUN apk add --no-cache curl build-base \
     && rm dockerize-alpine-linux-amd64-${DOCKERIZE_VERSION}.tar.gz
 
 # Install CKAN.
-ENV CKAN_VERSION 2.8.8
 
 RUN . ${VENV_DIR}/bin/activate \
     && pip install setuptools==36.1 \
-    && pip install -e "git+https://github.com/ckan/ckan.git@ckan-${CKAN_VERSION}#egg=ckan" \
+    && pip install -e "git+https://github.com/qld-gov-au/ckan.git@ckan-${CKAN_VERSION}#egg=ckan" \
```

Comment:
why is it our version compared to core.
site note: can't we have that as a variable also so that we can swap easier.

Reply:
It's possible, sure. Would you prefer the default to be ours or vanilla?

Reply:
so that we can test against our version as well as vanilla